tomviz/python/tomviz/itkutils.py

```diff
@@ -206,11 +207,11 @@ def get_python_voxel_type(dataset):
                 itkTypes.UC: int,
                 itkTypes.US: int,
                 itkTypes.UI: int,
-                itkTypes.UL: long,
+                itkTypes.UL: py2to3.long_type,
```

Comment:
Great, thanks, I wasn't about the ITK to python mapping. I know that int and long where pretty much the same, but was concerned that we might get tripped up by the subtle differences.